next-docs/pages/world/systems.mdx

```diff
@@ -1 +1,111 @@
 # Systems
+
+One of the design principles of MUD is to separate the state of the `World` from the business logic that it implements.
+We do this by having stateless `System` contracts with the functions that implement the business logic.
+These contracts are called through the `World`, and call back on the `World` when they need to read or modify state information, which is stored in [tables](./tables).
+
+<details>
+
+<summary>Detailed illustration</summary>
+
+![Detailed illustration](./world-table.svg)
+
+1. An external entity (contract or externally owned account) calls on the `World` a function called `myns_sys_fun`.
```

Comment:
We try to avoid abbreviations everywhere, should also do it in the docs
```suggestion
1. An external entity (contract or externally owned account) calls on the `World` a function called `namespace_system_function`.
```